src/meson.build

```diff
@@ -17,13 +17,24 @@ processor = executable('shotwell-graphics-processor',
                        dependencies: [gio, gdk, gee],
                        link_with: sw_graphics_processor)
 
+face_resources = []
```

Comment:
Technically, that's face_sources; resources is used for the resource files (ui, icons etc.)